app/controllers/admins/rewards_controller.rb

```diff
@@ -10,17 +10,20 @@ def show
 
     def new
       @reward = Reward.new
+      @possible_categories = Category.all
```

Comment:
@possible_categories is required for bulma form element to properly display possible options